test/cpp_extensions/rng_extension.cpp

```diff
@@ -63,8 +64,10 @@ static auto registry = torch::RegisterOperators()
       .op(torch::RegisterOperators::options()
         .schema("aten::random_(Tensor(a!) self, *, Generator? generator=None) -> Tensor(a!)")
         .impl_unboxedOnlyKernel<decltype(random_), &random_>(DispatchKey::CustomRNGKeyId));
+}
 
 PYBIND11_MODULE(TORCH_EXTENSION_NAME, m) {
+  m.def("registerOps", &registerOps);
```

Comment:
This change seems a bit weird. Could you explain in more details why this was necessary?

Reply:
This is done to enable the second rng extension(see [the next pr](https://github.com/pytorch/pytorch/pull/36633)) in the same `test/test_cpp_extensions_aot.py`

Reply:
Hmm, I am still a bit confused. So are you telling me that `rng_extension` and `csprng_extension` get loaded into the *same* Python process in the process of testing? What I would have expected is that we ought to load each extension in a separate Python process, so that various extensions don't interfere with each other (because, to me, if I have to independent extension tests, it should be valid for them to use dispatch keys in incompatible ways). I could certainly believe that this is true, but it doesn't seem *right*.

(I'm pushing on this because I really want these tests to use the "normal" API, involving static initializers. There should be a good reason to not use the normal API.)